compiler/luci-interpreter/src/kernels/DepthwiseConv2D.test.cpp

```diff
@@ -68,6 +68,67 @@ TEST(DepthwiseConv2DTest, Float)
   };
   EXPECT_THAT(extractTensorData<float>(output_tensor),
               ElementsAreArray(ArrayFloatNear(ref_output_data)));
+  EXPECT_THAT(extractTensorShape(output_tensor), ::testing::ElementsAreArray({1, 2, 1, 4}));
+}
+
+TEST(DepthwiseConv2DTest, Uint8)
+{
+  float kQuantizeTolerance = getTolerance(-127, 128, 255);
```

Comment:
Please be consistent with naming. We usually use snake_case for variable names.

Reply:
Oh, Sorry for make mistake. I will fix it.

Reply:
this variable is not used, so I remove it.